src/index.ts

```diff
@@ -4,3 +4,7 @@ export * from './useMount/useMount';
 export * from './useUnmount/useUnmount';
 export * from './useRegisterRef/useRegisterRef';
 export * from './useToggle/useToggle';
+export * from './useRefs/useRefs';
+export * from './useRefs/useRefs.types';
+export * from './useRefs/useRefs.util';
+export * from './trimEnd/trimEnd';
```

Comment:
Do we need to export this. IMO it's not really meant for public consumption.

Reply:
Not really, removed it from the exports